couchbase-operator/templates/_helpers.tpl

```diff
@@ -164,24 +164,53 @@ Generate cluster name from chart name or use user value
   {{ default (include "couchbase-cluster.fullname" .) .Values.cluster.name }}
 {{- end -}}
 
+
+{{/*
+Generate cluster spec
+*/}}
+{{- define "couchbase-cluster.spec" -}}
+{{- $spec := .Values.cluster -}}
+{{- $security := get $spec "security" -}}
+{{- $security := unset $security "password" -}}
+{{- $security := unset $security "username" -}}
+
```

Comment:
I forgot to set adminSecret here
```
{{- $security := set $security "adminSecret" (include "couchbase-cluster.admin-secret" .) -}}
```

Reply:
I was thinking I should extend the auto-test to wait for the cluster to spin up - can be done in parallel. At the moment it's just waiting for helm to install the chart which might pass but not actually work!